__tests__/app.test.js

```diff
@@ -81,6 +81,22 @@ describe('GET/api/articles/:article_id', () => {
         expect(res.body.msg).toBe('Bad request');
       });
   });
+  test('GET:200 sends article to client with data by id given and adds comment_count for total comments for article', () => {
+    return request(app)
+      .get('/api/articles/9')
+      .expect(200)
+      .then((res) => {
+        expect(res.body.article.comment_count).toBe(2);
+      });
+  });
+  test('GET:200 sends article to client with data by id, returns 0 for comment_count if there are no comments for article', () => {
+    return request(app)
+      .get('/api/articles/2')
+      .expect(200)
+      .then((res) => {
+        expect(res.body.article.comment_count).toBe(0);
+      });
+  });
```

Comment:
Nice testing! You could experiment with destructuring here, for example to get direct access to body in your then block then you could use {body} instead of res. Which would mean you can then remove res from your expext. You could then even take a step deeper, so to grab article you can do nested destructuring like so - {body: {article}}. This only works with keys that exist on the object, but you can get direct access through this and not need to chain dot notations